### ISSUE TITLE
wz rewards

### ISSUE THREAD

**Issue by Strangelunge1** [SmashMC-Development/Suggestions#1009]
there should wz rewards, like poketop. it makes the leaderboard actually worth something and it also brings in more competiton to wz

**Comment by WalkAway7** [comment]
The reward is wz itself, having the opportunity to go to a place where legends and ubs spawn all around is a good reward how it is.

**Comment by llnama** [comment]
there already is a /poketop

**Comment by oLimooo** [comment]
I like this suggestion. In the future I would love to see the warzone leader board be actually beneficial for us. Imo, the only thing it really does shows who is the "best player". The opportunities that Walk mentioned is true, part of warzone is getting the chance to catch rare legendaries in Warzone, however they are not always the best. Take example, last legendary I caught in warzone was a 58% UB. I just believe that the warzone leaderboard should be put into use instead of just show and tell.

**Comment by WalkAway7** [comment]
Yes that is a valid point. But the opportunity to get legendaries and ultra beasts and increased spawns anyway is a myth in regular pixelmon, you'd have to camp the same biome for ages until you actually get a legend or ultra beast. Plus it's not exactly expensive to go to warzone, for the chance to get something well above that price. And also the guarantee that you will be able to get many lvl 90+ pokemon. The point of the rep leaderboard in itself is meant to be a show and tell sort of thing anyway, on most servers it's pretty competitive to get the number one spot in wz. And it is as the feature says, it gives you reputation, not rewards
